### PR TITLE
Scope Requesting Feature

### PR DESCRIPTION
**Pull request by RomaRudyak**
In context of my application I need to access additional APIs of Google.
I add possibility to configure additional scope on platforms.

### CODE REVIEW

**Comment by RomaRudyak** [comment]
Hi @rdelrosario and @Pujolsluis,

in this pull request I have problem with retrieving of IdToken on android.
can you give hint.

as i can login and add additional scopes but still IdToken not returned

**Comment by Pujolsluis** [comment]
Hey @RomaRudyak ,

Great contribution!! Yes this feature overlaps with the one requested on issue #2 , @rdelrosario and me will be working this weekend on adding the AuthToken for Android since we have to look into it more, this is because it seems like it's not that straightforward as their iOS SDK. 

After we add your PR for the scopes and the AuthToken we will be releasing the first stable version of the plugin to the public, we will keep you posted 💪 .

**Comment by RomaRudyak** [comment]
@Pujolsluis cool, I already reserched things, and few threads point to use web app token one to configure to .RequestIdToken. but It all failed for me :(

**Comment by Pujolsluis** [comment]
Hey @RomaRudyak we just released a version that allows you to get the user token, we added a new optional parameter, for supporting the .requestTokenId method on the builder, send your Google Developer Console Android Client Id to that parameter and it should work.

Test it out and give us your feedback 💪

**Comment by RomaRudyak** [comment]
@Pujolsluis thank you fast release. But still no luck for me to receive AccessToken on Android.
Here is my [repo](https://github.com/RomaRudyak/xam-sdc/tree/977eb9305c03ce454abf5a954689820a7c2a6697)

**Comment by Pujolsluis** [comment]
Hey @RomaRudyak ,
The problem is that you have to send it as the 3rd parameter, since that's the one for the requestTokenId method. it should be like this GoogleClientManager.Initialize(this, null, <client id>);

**Comment by RomaRudyak** [comment]
I am using MvvmCross and there is some problem with MvxAppCompatActivity. I will dig in more and try find out what the problem is.
For standart AppCompatActivity all works

**Comment by RomaRudyak** [comment]
@Pujolsluis Found out the problem in my project. Created #12 issue

**Comment by Pujolsluis** [comment]
Goodjob! @RomaRudyak ,

We will review your changes and issues over the weekend 💪

File: GoogleClient/GoogleClient/Platforms/Android/GoogleClientManager.cs [hunk 1]
```diff
@@ -113,115 +136,112 @@ public void Logout()
...
+
+        private static void OnSignInSuccessful(GoogleSignInResult result)
         {
-            Debug.WriteLine(Tag + ": Connection to the client was suspended");
+            GoogleSignInAccount userAccount = result.SignInAccount;
+            GoogleUser googleUser = new GoogleUser
+            {
+                Name = userAccount.DisplayName,
+                Email = userAccount.Email,
+                Picture = new Uri((userAccount.PhotoUrl != null ? $"{userAccount.PhotoUrl}" : $"https://autisticdating.net/imgs/profile-placeholder.jpg"))
```
Comment (rdelrosario) [inline GoogleClient/GoogleClient/Platforms/Android/GoogleClientManager.cs]: Would be better to use a local placeholder or that the user specifies the placeholder.

Comment (RomaRudyak) [reply GoogleClient/GoogleClient/Platforms/Android/GoogleClientManager.cs]: @rdelrosario  Can we create Separate Issue with Feature label for this?
as it was already in project not related to this pull request.

**Pull request merged**